<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-10T16:25:56 IST" name="Default suite" finished-at="2025-03-10T16:26:24 IST" duration-ms="28085">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@687e99d8]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-03-10T16:25:56 IST" name="Default test" finished-at="2025-03-10T16:26:24 IST" duration-ms="28085">
      <class name="TestRunner.DemoNG">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.DemoNG@687e99d8]" started-at="2025-03-10T16:25:56 IST" name="setUpClass" finished-at="2025-03-10T16:25:58 IST" duration-ms="2546" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@6fff46bf]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@687e99d8]" started-at="2025-03-10T16:25:58 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-03-10T16:26:24 IST" duration-ms="25256" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check Login functionality of the Application"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Unknown"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [a edited Last Name] but found [test user benakba 1fakhreddine]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [a edited Last Name] but found [test user benakba 1fakhreddine]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinition.Sample2Scenario.user_verifies_the_profile_account(Sample2Scenario.java:30)
at âœ½.User verifies the Profile Account(file:///C:/Users/Admin/eclipse-workspace/PageFactoryEnabledMaven/src/test/resources/Feature/Sample3.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.DemoNG@687e99d8]" started-at="2025-03-10T16:26:24 IST" name="tearDownClass" finished-at="2025-03-10T16:26:24 IST" duration-ms="117" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.DemoNG -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
